SCClassLibrary/Common/Quarks/Git.sc

```diff
@@ -48,6 +48,22 @@ Git {
 		});
 		^nil
 	}
+	httpsString {
+		var giturl = this.url;
+		var hosturl, path;
+
+		if(giturl.notNil, {
```

Comment:
why? can be bitbucket or my own privately hosted gitlab. as telephon pointed out earlier.